app/locustio/jira/http_actions.py

```diff
@@ -70,10 +70,12 @@ def view_issue(locust):
     edit_allowed = fetch_by_re(params.edit_allow_pattern, content, group_no=0)
     locust.client.get(f'/secure/projectavatar?avatarId={project_avatar_id}', catch_response=True)
     # Assertions
+    if not(f'<meta name="ajs-issue-key" content="{issue_key}">' in content):
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')